### ISSUE TITLE
[MacOS 10.13.5]  WARNING: CocoaPods is not installed or is not configured properly.

### ISSUE THREAD

**Issue by NickIliev** [NativeScript/nativescript-cli#3695]
_From @turiguiliano88 on June 19, 2018 23:53_

This message prompted out after installing nativescript on MacOS 10.13.5 and run "sudo tns doctor". Therefore I am not able to run "sudo tns run ios". I keep emphasizing sudo here because I just can not run without it. I did follow option "config with local build" but it doesn't solve the problem. My Xcodeproj version is 1.5.9, node v8.11.3.

_Copied from original issue: NativeScript/NativeScript#5973_

**Comment by NickIliev** [comment]
@turiguiliano88 do you have CocoaPods installed on your Mac?
```
pod --version
```

If not, please install them with
```
sudo gem install cocoapods
```

**Comment by NickIliev** [comment]
_From @turiguiliano88 on June 20, 2018 7:12_

@NickIliev Hi, I had already installed CocoaPods before. Version 1.5.3.
During "sudo tns doctor" there were also messages:
✔ CocoaPods update is not required.
✔ Your current CocoaPods version is newer than 1.0.0.

**Comment by dtopuzov** [comment]
Hi @turiguiliano88

After installing cocoa pods you should also run `pod setup` (without sudo).
It was missing in docs, but we fixed it with this [PR](https://github.com/NativeScript/docs/pull/1181/files#diff-734ce2e1c5561d9239f5b049357f8429R91) and it should be on released on [docs site](https://docs.nativescript.org/start/ns-setup-os-x) in next days.

**Comment by turiguiliano88** [comment]
@dtopuzov Hi. As you recommended, I tried to run `pod setup` and it didn't work, the warning message still there. I've also tried to uninstall completely and reinstall cocoapods from this link [https://gist.github.com/derekli66/45406e2f94a2d8056e34](url) but it doesn't work either. Do you have other suggestions?

**Comment by turiguiliano88** [comment]
The issues was generated because of permission. After I set permisson `sudo chmod -R 777 ~/.local/` everything is running properly. Thanks for all the help btw 👍

**Comment by thomascmost** [comment]
I'm still experiencing this. Can you explain the permission issue?

**Comment by thomascmost** [comment]
CocoaPods 1.5.3

I've run `pod setup`
`tns doctor` says it's installed
`tns run [any]` says `Cocoa pods is not installed or is not configured properly`

**Comment by rosen-vladimirov** [comment]
Hey @thomascmost , 
Can you try running `pod repo update` in your command line and build your project afterwards?

**Comment by thomascmost** [comment]
No dice. Same error

**Comment by thomascmost** [comment]
no, tns doctor says everything is fine, even run without `sudo`

**Comment by rosen-vladimirov** [comment]
Hey @thomascmost , can you send me the exact error when running `pod repo update`. It looks like your cocoapods repository is messed up. In similar case I just delete  my `~/.cocapods`  directory and run `pod setup`, so you may give it a try.

**Comment by tibawatanabe** [comment]
I was facing the same problem until today. I've executed every instruction I've found but to no avail...

These are the steps I've followed that solved this issue:

1. (I'm using RVM) Remove all ruby versions and ensure that everything was deleted (gems, bin, etc)
    - I've found out that simply uninstalling the gems was not deleting everything, making the `tns doctor` detect something that was not there
2. Ensure that the `tns doctor` is coherent with the machine dependencies without installing anything - at this point, it should detect that both, cocoapods and xcodeproj were missing.
3. Reinstall ruby
4. Run `tns doctor` and use its script to install all ruby dependencies again
